src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -156,7 +160,11 @@ public string GetBaggageItem(string key)
         public Activity(string operationName)
         {
             if (string.IsNullOrEmpty(operationName))
-                throw new ArgumentException($"{nameof(operationName)} must not be null or empty");
+            {
+                NotifyError(new ArgumentException($"{nameof(operationName)} must not be null or empty"));
```

Comment:
I'm assuming we're not putting these messages in resources now because we only expect these to be seen in a debugger breaking on first-chance exceptions?